src/misc/news/items/ReferralLinkViewer.ts

```diff
@@ -36,11 +37,7 @@ export class ReferralLinkViewer implements Component<ReferralLinkAttrs> {
 			label: "referralLink_label",
 			value: referralLink,
 			injectionsRight: () => this.renderButtons(referralLink),
-			helpLabel: () =>
-				ifAllowedTutaLinks(locator.logins, InfoLink.ReferralLink, (link) => [
-					m("span", lang.get("moreInfo_msg") + " "),
-					m("span.text-break", [m(`a[href=${link}][target=_blank]`, link)]),
-				]),
+			helpLabel: () => ifAllowedTutaLinks(locator.logins, InfoLink.ReferralLink, (link) => [m(MoreInfoLink, { link: link as InfoLink })]),
```

Comment:
we should change  type for `ifAllowedTutaLinks`

should be:

```ts
export function ifAllowedTutaLinks(logins: LoginController, linkId: InfoLink, render: (linkId: InfoLink) => Children): Children | null
```

then you don't need a cast

Reply:
Understood, will do!